packages/react/src/useSynthql.ts

```diff
@@ -25,7 +25,15 @@ export function useSynthql<
 ): UseQueryResult<QueryResult<DB, TQuery>> {
     const { endpoint, requestInit } = useSynthqlContext();
 
-    const enrichedEndpoint = `${endpoint}/${query.from}`;
+    const hash = hashQuery(query);
```

Comment:
you shouldn't calculate the hash on the react side, you should calculate it on the query builder.